src/ProjectModule/Components/Home/Home.tsx

```diff
@@ -1,7 +1,185 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from 'react'
+import styles from "./Home.module.css"
+import { Doughnut } from 'react-chartjs-2';
+import axios from 'axios';
+import {Tooltip,Title,ArcElement,Legend,Chart as ChartJS} from "chart.js"
+import { baseUrl } from '../../../Constants/Components/Urls';
+import { AuthContext } from '../../../Context/Components/AuthContext';
+ChartJS.register(Tooltip,Title,ArcElement,Legend);
+
 
 export default function Home() {
+const[taskToDo,setTaskToDo]=useState(0);
+const[taskProgres,setTaskProgres]=useState(0);
+const[taskDone,setTaskDone]=useState(0);
+const[userActive,setUserActive]=useState(0);
+const[userInActive,setUserInActive]=useState(0);
+const{userRole}=useContext(AuthContext)
+
+
+const getTasksCount=async()=>{
+  try {
+    const token=localStorage.getItem("userToken")
+    const response=await axios.get(`${baseUrl}/Task/count`,{headers:{Authorization:token}}) 
+    setTaskToDo(response.data.toDo);
+    setTaskProgres(response.data.inProgress);
+    setTaskDone(response.data.done);
+  
+    console.log(response?.data);
+   
+  } catch (error) {
+    console.log(error)
```

Comment:
🛠️ You might want to handle error cases in axios requests and provide feedback to users.